UI/win-update/updater/CMakeLists.txt

```diff
@@ -1,13 +1,18 @@
-project(updater)
+cmake_minimum_required(VERSION 3.24...3.25)
+
+legacy_check()
 
 option(ENABLE_UPDATER "Build with Windows updater" ON)
 
 if(NOT ENABLE_UPDATER)
-  message(STATUS "OBS:  DISABLED   Windows updater")
+  target_disable_feature(obs "Windows updater")
   return()
 endif()
 
-find_package(zstd)
+if(NOT DEFINED STATIC_ZLIB_PATH OR "${STATIC_ZLIB_PATH}" STREQUAL "")
+  message(AUTHOR_WARNING "STATIC_ZLIB_PATH not set, windows updater disabled")
+  return()
+endif()
```

Comment:
That specific file wasn't fully updated because it also isn't supposed to take effect already - I've updated it accordingly but the "actual" update will happen with the Windows-variation of this PR.